syncd/syncd_flex_counter.cpp

```diff
@@ -58,6 +59,14 @@ FlexCounter::RifCounterIds::RifCounterIds(
     SWSS_LOG_ENTER();
 }
 
+FlexCounter::BufferPoolCounterIds::BufferPoolCounterIds(
+        _In_ sai_object_id_t bufferPool,
+        _In_ const std::vector<sai_buffer_pool_stat_t> &bufferPoolIds):
+    bufferPoolId(bufferPool), bufferPoolCounterIds(bufferPoolIds)
+{
+    SWSS_LOG_ENTER();
```

Comment:
why this is needed if empty ?

Reply:
This is constructor. We use data member initialization syntax